README.md

```diff
@@ -102,6 +102,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 
 ## S - U
 - [Sastodeal - Be Genuine](https://www.sastodeal.com/) | Tripureshwor, Kathmandu | Provide e-commerce services for businesses to connect to it's customers online
+- [Segonatech](https://segonatech.com/) | Battisputali, Kathmandu | A refined and enthusiastic social affair of organizers, fashioners, wander chiefs, researchers and experts. Every client we work with transforms into a bit of the gathering. Together we stand up to the challenges and laud the triumphs.
```

Comment:
@ngima The description is too long here. Also, can we make it a single sentence?

And please remove the period `.` at the end.